Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.2
+MAINTAINER dengnan@google.com vmarmol@google.com vishnuk@google.com jimmidyson@gmail.com
+
+WORKDIR /gopath/src/github.com/google/cadvisor
+COPY    . /gopath/src/github.com/google/cadvisor
+
+RUN apk add --update --repository=http://dl-4.alpinelinux.org/alpine/edge/community/ -t build-deps go sqlite-dev btrfs-progs-dev bash linux-headers make git gcc g++ && \
```

Comment:
Since this is run after the copy, docker won't cache any of these commands.  Does it make sense to split out the commands that do not depend on the src code and run them first?